packages/ecc-client-elixir-drs-filer/src/API/Object/drsAPI.ts

```diff
@@ -28,4 +36,33 @@ const postObject = async (baseURL: string, objectData: object) => {
   }
 };
 
-export { postObject };
+/**
+ *This method deletes a specific object
+ * @param id ID of the drs obeject to be deleted
+ */
+const deleteObject = async (baseURL: string, id: string) => {
+  const url = `${baseURL}/objects/${id}`;
+  try {
+    const response = await fetch(url, {
+      method: "DELETE",
+    });
+
+    if (!response) {
```

Comment:
consider some more robust error handling like checking the status text, as checking for just the response object won't catch many errors that may occur.

Also consider this for the other `postObject` function.